nilearn/datasets/neurovault.py

```diff
@@ -909,6 +914,8 @@ def _print_if(message, level, threshold_level, with_traceback=False):
         if `message` is printed, also print the last traceback.
 
     """
+    # TODO replace by the nileanr logger,
+    # that does almost exactly the same thing!
     if level > threshold_level:
```

Comment:
Possible to remove an entire private function from this and only rely on the nilearn logger that does the same thing.